routing.md

```diff
@@ -280,11 +280,27 @@ Why is such indirection useful?
 
 These last two characteristics are the foundation of mix networking feature for DIDComm. That feature is the subject of a different RFC; here we only note the existence of the optional feature. 
 
+### Sender Forward Process
+
+1. Sender Constructs Message
+2. Sender Encrypts Message to recipient(s)
+3. Wrap Encrypted Message in Forward Message for each Routing Key.
+4. Transmit to `serviceEndpoint` in the manner specified in the [transports] section.
+
+### Mediator Process
+
+_Prior to using a Mediator, it is the recipient's responsibility to coordinate with the mediator. Part of this coordination informs them of the `to` address(es) expected, the endpoint, and any Routing Keys to be used when forwarding messages. That coordination is out of the scope of this spec._
+
+1. Receives Forward Message
+2. Retrieves Service Endpoint and Routing Keys as pre-configured by recipient (`to` attribute).
+3. Wrap attached `payload` message once per Routing Key
+4. Transmit to Service Endpoint in the manner specified in the [transports] section.
+
 ### DID Document Keys
 
 All keys declared in the DID Document's `keyAgreement` section should be used to encrypt messages. The details of key representation are described in the [Public Keys section of the DID Core Spec](https://www.w3.org/TR/did-core/#public-keys).
```

Comment:
suggest a tweak: "should be used to encrypt messages" --> "should be used as recipients when encrypting messages"